src/comment.ts

```diff
@@ -27,14 +29,18 @@ enum TypeOfComments {
 type Mention = { rule: string; mentions: string[]; URL: string };
 
 const LINE_BREAK = '<br/>';
+const USE_HERALD_ACTION_TAG_REGEX = /^<!-- USE_HERALD_ACTION (.*) -->$/;
```

Comment:
why do you need this regex? if the comment could simply be `<!-- USE_HERALD_ACTION -->`

Reply:
resolving after discussion offline